src/components/AboutUs/Cards.js

```diff
@@ -14,15 +14,17 @@ import shiny3 from '../../images/about-us/How-are-we-different-section/shiny3.sv
 
```

Comment:
Imports should be separated by new line between packages and internal

Reply:
done